OpenTreeMap/src/OTM/Cells/OTMDetailCellRenderer.m

```diff
@@ -323,9 +327,22 @@ - (void)buildCell:(OTMDetailTableViewCell *)detailcell fromData:(NSDictionary *)
                                              );
 
     detailcell.unitLabel.text = _formatter.label;
+    self.wasCleared = NO;
     self.inited = YES;
 }
 
+-(NSDate *)dateFromOTMDateString:(NSString *)dateString
+{
+    NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+    [dateFormatter setDateFormat:OTMEnvironmentDateStringShort];
+    NSDate *newDate=[dateFormatter dateFromString:dateString];
```

Comment:
Might want to format with spaces around equals sign here

Reply:
Deferring this whitespace fix to to avoid spending time rebasing and retesting.